### ISSUE TITLE
Fix node approval modal

### ISSUE THREAD

**Issue by faboweb** [cosmos/voyager#638]
Currently:
![image](https://user-images.githubusercontent.com/5869273/38732139-766c2944-3f1d-11e8-8d74-080785652963.png)

**Comment by jbibla** [comment]
- css issues
- also discuss new ux

**Comment by NodeGuy** [comment]
What is this blocked by?

**Comment by faboweb** [comment]
This is currently never appearing to the user as the process for this is commented out.
As discussed this is blocked on defining how this process actually will look like. We don't need to work on things that are not used right now.

**Comment by jbibla** [comment]
i think our idea for this was to implement some kind of "trust status" component so that users would not have to do this. is there a record / ticket for that feature somewhere? @cosmos/cosmos-ui ?

related to #100 for the record.

**Comment by faboweb** [comment]
Please fo not discuss any node approval process in here. This is just about fixing css.

**Comment by jbibla** [comment]
ok. where should we discuss node approval process?

fixing the css is blocked?

**Comment by faboweb** [comment]
@jb new epic here https://github.com/cosmos/voyager/issues/1222

**Comment by faboweb** [comment]
> it's implying an inaccurate priority

doesn't the `for later` flag tell us, that it is not a priority? Still, I don't mind removing it.

**Comment by jbibla** [comment]
- this ticket is for fixing a css bug on the existing modal (which may be deprecated in the future if we implement #1222 well)
- #1222 will require more conversation and design

so this can be `for later` IMO

**Comment by NodeGuy** [comment]
It's for fixing a CSS bug on the existing modal but that modal is never shown, correct?

**Comment by faboweb** [comment]
@NodeGuy correct

**Comment by NodeGuy** [comment]
I removed the `bug` label because it's not user-facing.

**Comment by fedekunze** [comment]
is this fixed ?

**Comment by jbibla** [comment]
probably not - but we can close because we don't show this modal/
